custom_components/easee/easee.py

```diff
@@ -46,7 +46,74 @@ async def get_consumption_between_dates(self, from_date, to_date):
         return float(value)
 
     async def start(self):
-        return await self.session.post(f"/api/chargers/{self.id}/commands/start_charging")
+        """Start charging session"""
+        return await self.easee.post(f"/api/chargers/{self.id}/commands/start_charging")
+
+    async def pause(self):
+        """Pause charging session"""
+        return await self.easee.post(f"/api/chargers/{self.id}/commands/pause_charging")
+
+    async def resume(self):
+        """Resume charging session"""
+        return await self.easee.post(
+            f"/api/chargers/{self.id}/commands/resume_charging"
+        )
+
+    async def stop(self):
+        """Stop charging session"""
+        return await self.easee.post(f"/api/chargers/{self.id}/commands/stop_charging")
+
+    async def toggle(self):
+        """Toggle charging session start/stop/pause/resume """
+        return await self.easee.post(
+            f"/api/chargers/{self.id}/commands/toggle_charging"
+        )
+
+    async def get_basic_charge_plan(self):
+        """Get and return charger basic charge plan setting from cloud """
+        return await self.easee.get(
+            f"/api/chargers/{self.id}/commands/basic_charge_plan"
+        )
+
+    async def set_basic_charge_plan(
+        self, id, chargeStartTime, chargeStopTime, repeat=True
```

Comment:
What id is sent in here?

Reply:
Good question. The API has this id for set/get plan, but I believe I had to use the chargerID. I'll test it again, don't remember exactly, but I think I managed to set, get and delete the plan. Perhaps they are somehow preparing for multiple plans and didn't complete it yet. I'll test and come back to this.